README.md

```diff
@@ -50,7 +50,7 @@ while n < data.len() {
     let res = inflater.update(&data[n..]);
     if let Ok((num_bytes_read, result)) = res {
         n += num_bytes_read;
-        out.extend(result);
+        out.extend(result.iter().map(|v| *v));
```

Comment:
Probably, couldn’t remember what was the most elegant way to solve it back then…